### ISSUE TITLE
Questions: why live snippets cannot be used in inline math env

### ISSUE THREAD

**Issue by hktkzyx** [tecosaur/LaTeX-Utilities#41]
The live snippets don't function when in inline math env, like $...$.

**Comment by tecosaur** [comment]
This comes down to a matter of performance. For `\( ... \)` inline math. Using `|` to show the cursor position, if you see `... \( ... | ` you _know_ that the cursor is in a maths block. If you have `... $ ... |` you don't _know_ without scanning the entire document up to the cursor position. Consider the following example
```tex
...
not math?  $ math? $ not math? $ math? $ not math? $
math? $ not math? $ math? $ not math? $ math? |
```
Since this determination is done every keystroke, I don't think this is worth the effort. Also if you see [this](https://tex.stackexchange.com/questions/510/are-and-preferable-to-dollar-signs-for-math-mode#comment2607_513 ) you'll see that `$ ... $` is considered TeX code, not LaTeX, and probably won't be supported in LaTeX3.

**TLDR; Don't use `$ ... $`, use `\( ... \)`.**
